CMakeLists.txt

```diff
@@ -35,31 +35,21 @@ CMAKE_MINIMUM_REQUIRED( VERSION 3.8.0 )
 SET( CMAKE_SKIP_RPATH TRUE )
 
 ## project name & version
-PROJECT( partio )
-SET( ${PROJECT_NAME}_MAJOR_VERSION 0 )
-SET( ${PROJECT_NAME}_MINOR_VERSION 1 )
-SET( ${PROJECT_NAME}_PATCH_LEVEL 0 )
-include(GNUInstallDirs)
+PROJECT( partio LANGUAGES CXX VERSION 0.1.1 )
```

Comment:
@davvid I bumped this to `0.1.1` for all the fixes. Could you please tag a new release so we can add it in Gentoo and remove all the patches?

Reply:
Hi @SoapGentoo I just tagged partio `v1.5.0`.  This includes a merge of our internal changes so that we're 1:1 identical with the open source version now.  Let me know if I missed anything and we can fix and tag a new version as needed.